Objects/obmalloc.c

```diff
@@ -1286,6 +1423,33 @@ _PyObject_Realloc(void *ctx, void *p, size_t nbytes)
     return bp ? bp : p;
 }
 
+
+static void *
+_PyObject_AlignedAlloc(void *ctx, size_t alignment, size_t size)
+{
+    assert(check_alignment(alignment));
+    if (!check_alignment(alignment)) {
+        return NULL;
+    }
+
+    if (alignment <= ALIGNMENT) {
+        return _PyObject_Alloc(0, ctx, 1, size);
```

Comment:
Move check_alignment(alignment) only in this path?

Reply:
Ah yes, it makes sense. Done.

I also merged master into this PR.